### DISCUSSION TITLE
Cross Browser and different Screen size

### DISCUSSION THREAD

**Discussion by pst37** [lost-pixel/lost-pixel#292] [General]
Hey I just wanted to check do lost-pixel have support for cross browser with different screensize.

**Comment by d-ivashchuk** [comment]
Hey @prafullsingh37! Different viewports yes, different browsers nope. You could only choose which browser to run on but not have both in single execution. Technically you could fake it by running two GitHub actions - 1 for Mozilla 1 for chrome, that's what we support right now. Tell me if you want to try and figure it out - I would be happy to support you!

**Reply by pst37** [reply]
Could you help me on the different viewport ? I tried searching in documentation not able to find out ...

**Reply by d-ivashchuk** [reply]
https://github.com/lost-pixel/lost-pixel/pull/255#issuecomment-1670035565

Look here, we are intending to merge the breakpoint shots into the main version of LP this week

**Comment by d-ivashchuk** [comment]
@prafullsingh37 breakpoint shots are merged already and you could use them as suggested above but in the latest version of lost pixe l
